homeassistant/components/shopping_list/__init__.py

```diff
@@ -190,7 +190,7 @@ def async_handle(self, intent_obj):
 
         response = intent_obj.create_response()
         response.async_set_speech(f"I've added {item} to your shopping list")
-        intent_obj.hass.bus.async_fire(EVENT)
+        intent_obj.hass.bus.async_fire("shopping_list_item_added", {"item_added": item})
```

Comment:
I suggest that instead of placing the event directly, you create global variables (or a dictionary) to make changes easier. For example, change `EVENT` to `EVENT_UPDATE`, and `EVENT_ITEM_ADD`.